src/Build.UnitTests/TargetsFile_Test.cs

```diff
@@ -434,7 +435,7 @@ public void AnyCPU40ExeProjectIsNot32BitPreferred()
 
         /// <summary>
         /// Check that with an explicit platformtarget of anycpu, exe type assemblies that are 
-        /// targeting .NET 4.0 do not get forced to anycpu32bitpreferred by default. 
+        /// targeting .NET 4.8 do not get forced to anycpu32bitpreferred by default. 
         /// </summary>
         [Fact]
         public void ExplicitAnyCPU40ExeProjectIsNot32BitPreferred()
```

Comment:
This test feels TFM-specific. How did you validate that it was safe to update?

Reply:
I can't leave it at 4.0, since that wouldn't be on the box, so I don't think I have a choice unless I remove it entirely.

Reply:
Yes, that's the question at hand: was this actually specific to 4.0 and thus can now be removed or disabled unless the specific build machine has that SDK installed?

Reply:
This came in with the initial code commit, so I don't have anything specific I can reference. From reading the comment, I would suspect the important part is that it isn't being pushed to anycpu32bitpreferred when it's supposed to be AnyCPU. That also aligns with the only assert in the test. I don't think the TF actually matters.

Reply:
> This came in with the initial code commit, so I don't have anything specific I can reference.

That doesn't need to be a blocker for you since you're inside the firewall. This was implemented in (internal link) https://vstfdevdiv/DevDiv2/DevDiv/_versionControl/changeset/320633/ with the message

> Some class library projects were failing with an error regarding AnyCPU32BitPreferred not being an allowed target type for library projects. This was due to two issues:
> 
> * The managed compilers were only remapping AnyCPU to AnyCPU32BitPreferred when AnyCPU was explicitly set, which meant that projects that did not explicitly set PlatformTarget to AnyCPU would silently fail to remap and thus silently succeed in the build.
> * The managed and Jupiter targets were setting the Prefer32Bit flag to true across the board for 4.5 projects, when they should have been only doing so if the project had a supported outputtype (exe, winexe, or appcontainerexe).
>
> This shelveset fixes both issues: the managed compilers now remap to AnyCPU32BitPreferred when the PlatformTarget is empty, as well as when it is explicitly AnyCPU; and the managed and Jupiter targets now only set Prefer32Bit to true if the project is one of the supported output types.

So it looks like the best fix would be to change the comment and possibly test names to `4.0+`.